src/web/nextui/src/app/eval/ResultsView.tsx

```diff
@@ -235,15 +245,62 @@ export default function ResultsView({
     setAnchorEl(event.currentTarget);
   };
 
+  const [evalIdCopied, setEvalIdCopied] = React.useState(false);
+
+  const handleEvalIdCopyClick = async () => {
+    if (!evalId) {
+      return;
+    }
+    await navigator.clipboard.writeText(evalId);
+    setEvalIdCopied(true);
+    setTimeout(() => {
+      setEvalIdCopied(false);
+    }, 1000);
+  };
+
   return (
     <div style={{ marginLeft: '1rem', marginRight: '1rem' }}>
-      <Box mb={2} sx={{ display: 'flex', alignItems: 'center' }}>
-        <Heading variant="h5" sx={{ flexGrow: 1 }}>
+      <Box mb={2} className="eval-header">
+        <Typography variant="h5" sx={{ display: 'flex', alignItems: 'center' }}>
           <span className="description" onClick={handleDescriptionClick}>
             {config?.description || evalId}
-          </span>{' '}
-          {config?.description && <span className="description-filepath">{evalId}</span>}
-        </Heading>
+          </span>
+        </Typography>
+        {config?.description && evalId && (
+          <>
+            <Chip
+              size="small"
+              label={
+                <>
+                  <strong>ID:</strong> {evalId}
+                </>
+              }
+              sx={{
+                ml: 1,
+                opacity: 0.7,
+                cursor: 'pointer',
+              }}
+              onClick={handleEvalIdCopyClick}
+            />
+            <Snackbar
+              open={evalIdCopied}
+              autoHideDuration={1000}
+              onClose={() => setEvalIdCopied(false)}
+              message="Eval id copied to clipboard"
+            />
+          </>
+        )}
+        {author && (
+          <Chip
+            size="small"
+            label={
+              <>
+                <strong>Author:</strong> {author}
```

Comment:
It would be great if there was a little hover tooltip that told people to set the author using the cli if it's not set.